components/ide/jetbrains/image/status/main.go

```diff
@@ -56,6 +57,45 @@ type LaunchContext struct {
 	wsInfo           *supervisor.WorkspaceInfoResponse
 }
 
+func findRiderSolutionFile(root string) string {
+	slnRegEx := regexp.MustCompile(`^.+\.sln$`)
+	projRegEx := regexp.MustCompile(`^.+\.csproj$`)
+
+	var slnFiles []string
+	var csprojFiles []string
+
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			log.Fatal(err)
+		} else if slnRegEx.MatchString(info.Name()) {
+			slnFiles = append(slnFiles, info.Name())
+		} else if projRegEx.MatchString(info.Name()) {
+			csprojFiles = append(csprojFiles, info.Name())
+		}
+		return nil
+	})
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
it will exit the process completely and supervisor will restart it over, going in never ending failure loop

Reply:
Maybe you wanted to use `log.Error`?

Reply:
Fixed 🙏